src/components/dls/Toggle/Toggle.module.scss

```diff
@@ -0,0 +1,42 @@
+@use "src/styles/theme";
+
+$size: var(--spacing-medium);
+
+.toggle {
+  cursor: pointer;
+  height: $size;
+  width: calc(2.5 * $size);
+
+  padding: calc(0.3 * var(--spacing-xsmall));
+
+  transition: 100ms linear;
+  border-radius: var(--border-radius-pill);
+  display: flex;
+  align-items: center;
+  background-color: var(--color-text-faded);
+}
+
+.handle {
+  background-color: var(--color-background-default);
+  @include theme.dark {
+    background-color: var(--color-background-inverse);
+  }
+
+  border-radius: 50%;
+
+  display: inline-block;
+  height: $size;
+  width: $size;
+  transition: transform var(--transition-fast) ease;
+}
+
+.checked > .handleOn {
+  transform: translateX(150%);
```

Comment:
nit: `150%` should be a variable since it's used twice

Reply:
the other one is `-150%`,I think it's okay since they're very close to each other.

Reply:
No strong opinion but I am thinking if we are going to refactor this or change the value, we will need to apply the change in 2 places or we might miss one of them